src/css/Timer.scss

```diff
@@ -1,11 +1,14 @@
 @import 'variables';
 
+$timer-height: 60px;
```

Comment:
Is this even used anywhere? You can change the height of the timer in the layout settings, so we probably do a local style anyway and don't use this I would assume.

Reply:
Oh wait, this is for the font-size? Then idk maybe this is used?